torch/_inductor/tiling_utils.py

```diff
@@ -196,7 +231,6 @@ def zip_equal(it1: Iterable[T], it2: Iterable[U]) -> Iterator[tuple[T, U]]:
         """
         if len(it1) != len(it2):
             raise ValueError(f"Lengths differ: {len(it1)} != {len(it2)}")
-        return zip(it1, it2)
```

Comment:
Whoops?

Reply:
i think the inductor gpu tests are cuda 3.10.. will udpate, good catch